metabase/metabase
# fix(sdk): ability to save questions in interactive question

Closes https://github.com/metabase/metabase/issues/48788

### Description

Implement the UI to save questions in the `InteractiveQuestion` component

### How to verify

- Add the `InteractiveQuestion` component to a page
- Add the `isSaveEnabled={true}` prop to your component
- The "save" button should be **disabled** by default
- Make change to your question a bit (e.g. removing filters, or drag to zoom into question, or add filters)
- The "save" button should be **enabled**
- Click on "Save"
- A save dialog should appear
- You should be able to save the question now

### Demo

The "Save" button now appears.

![CleanShot 2567-10-18 at 07 41 17@2x](https://github.com/user-attachments/assets/c65c3a74-b671-4b11-ab39-c4787773317c)

### Checklist
